### ISSUE TITLE
bug: drawer not receiving focus on open

### ISSUE THREAD

**Issue by Kinbaum** [emilkowalski/vaul#108]
If you load up any of the sandboxes from the readme and trigger the drawer to open, the first tab-able element within the drawer does not receive focus. This is actually breaking the a11y radix gives us. The focus is actually remaining on the trigger. Any tab-able elements after the trigger will receive focus first as you tab around before eventually landing in the drawer with the focus trap.

Here is a video of the behavior as well as a sandbox which is a direct fork of the example:
https://codesandbox.io/p/sandbox/drawer-with-scale-forked-6ss9nx?file=/app/page.tsx:8,10

https://github.com/emilkowalski/vaul/assets/6590356/a321586f-b746-4cba-b5f4-07b4bfe9bafd

**Comment by emilkowalski** [comment]
Fixed in #115

**Comment by chelscodes** [comment]
Also running into this with a drawer that has inputs and curious if there's a fix.
